assets/scripts/store/slices/street.js

```diff
@@ -66,6 +66,9 @@ const streetSlice = createSlice({
       }
     },
 
+    updateAnalytics (state, action) {
+      state.showAnalytics = action.payload
```

Comment:
I'm wondering if it's worthwhile to plan ahead for an analytics object to exist at this top level, so the format is:

```
state: {
  analytics: {
    segmentLabels: false
    // ... reserve for future settings
  }
}
```

One thing I'd wish we had time to fix was how we stored buildings as an object instead of individual property names (e.g. `leftBuilding`, `leftBuildingHeight` etc.), so we might want to get in front of that regret here.

Reply:
I think we should probably keep this at the same level as `width` and other top-level street settings, we will attach street capacity to each segment as discussed. Users could then put 0 capacity to hide capacity for a specific segment. Does that seem reasonable?

Reply:
I don't mean individual capacity should be stored here, I mean that there could be other analytics-related toggles that might exist in the future.

OK, I'll leave this alone for the moment because we don't have those additional toggles yet.

When it comes to toggling whether to show/hide capacity on individual segments, a input of zero is not actually the same as a display toggle. I get that it seems like a valid shortcut but this has bitten me in the past, and we shouldn't assume that's going to semantically mean the same thing. That's a different subject for discussion, however.